calculations/common/queryLanguage.lp

```diff
@@ -93,6 +93,16 @@ showField(Key, Field) :-
     link(Source, Destination, LinkType),
     showAll(Destination).
 
+% notifications
+
+#show notification(Key, TestSuite, TestCase, Message) :
```

Comment:
Why `TestSuite`, `TestCase`? Terminology doesn't make sense (this was already in earlier changes).

Reply:
@henryhaverinen any comments? As these are also connected to policy failures/successes, maybe they could be called "PolicyGroup" and "Policy" or something similar?

Reply:
We have been reusing terms from JUnit, where test cases are organised in test suites. This probably makes sense for the policyCheckFailures and policyCheckSuccesses, but I agree that it is less natural with the notifications.

Notifications are advice or hints, so they are not necessarily related to policy. Should we keep the test suite and test case terms as they are for policy checks, but try to come up with better terms for notifications?  For example Category and Title? 

On the other hand, Category and Title would probably be more natural also for the policy checks.

Reply:
We don't have a lot of policy check implementations in the modules yet, so I propose that we switch to Category and Title also for policy checks. (Doesn't have to be a part of this PR)

Reply:
It's done now, both Notifications and Policy checks now have category/title in use. I also updated the feature/henryhaverinen/INTDEV-539-notifications branch in unified-sdl repo